app/models/Job.js

```diff
@@ -1,16 +1,95 @@
-import listJobs from '../data/json/list_jobs';
-import { containsAny } from '../utils/math';
+import BaseModel from './BaseModel'
+import uuidv4 from '../utils/uuidv4';
+import arrayUtil from '../utils/array_util';
+import jobs from '../data/json/jobs.json';
+import Video from './Video';
+import School from './School';
+
+const MODEL = 'Job'
 
 export default class Job {
+  static seedData = () => {
+    jobs.map((job) => {
+      this.create(job)
+    })
+  }
+
+  static create = (data) => {
+    BaseModel.create(MODEL, this.getFormattedData(data), 'modified')
+  }
+
+  static getAll = () => {
+    return BaseModel.getAll(MODEL)
+  }
+
+  static findAllByJobCluster = (jobClusterId) => {
+    return [...BaseModel.findByAttr(MODEL, { job_cluster_id: `'${jobClusterId}'` })]
+  }
+
+  static findById = (id) => {
+    return BaseModel.findByAttr(MODEL, { id: `'${id}'` })[0]
+  }
+
   static findByCode = (code) => {
-    return listJobs.filter(x => x.code == code)[0];
+    return BaseModel.findByAttr(MODEL, {code: `'${code}'`})[0]
+  }
+
+  static findAllByPersonalityTypes = (types) => {
+    let jobs = []
+    types.map(type => {
+      jobs = [...jobs, ...BaseModel.containsByAttr(MODEL, 'personality_type', `'${type}'`)]
+    })
+    return arrayUtil.filterDuplicate(jobs, 'id')
+  }
+
+  static getVideosById = (jobId) => {
+    const job = this.findById(jobId)
+    if (!job) return []
+
+    let videos = []
+    job.videos.map(videoId => {
+      const video = Video.findById(videoId)
+      if (!!video)
+        videos.push(video)
+    })
+    return videos
+  }
```

Comment:
```
static getVideosById = (jobId) => {
    const job = this.findById(jobId)
    if (!job) return []

    let videos = []
    job.videos.map(videoId => {
      const video = Video.findById(videoId)
      if (!!video)
        videos.push(video)
    })
    return videos
  }
```
Can we use filter instead of finding one by one